service/plugins/perf/static/js/perf.js

```diff
@@ -102,14 +102,27 @@ mciModule.controller('PerfController', function PerfController($scope, $window,
   $scope.Math = $window.Math;
   $scope.conf = $window.plugins["perf"];
   $scope.task = $window.task_data;
-  $scope.tablemode = "maxthroughput";
+  $scope.tablemode = 'maxthroughput';
+  $scope.threadLevelsRadio = {
```

Comment:
Where is this changed in the UI? I wasn't able to find it

Reply:
Oh, that's a mistake. I'm working on two features at the same time, and it shouldn't be in this PR at all.